app/scripts/TrackRenderer.js

```diff
@@ -1486,6 +1487,18 @@ class TrackRenderer extends React.Component {
       }
     }
 
+    if (
+      track.options &&
+      track.options.selectRows &&
+      track.options.selectRowsAggregationMethod === 'server' &&
```

Comment:
We shouldn't be checking for track-specific options in `TrackRenderer`. You can set dataConfig directly in the `HorizontalMultivecTrack` instead of checking every track for these options here.